### PR TITLE
Feature/select

### PR DESCRIPTION
**Pull request by mahsa-ameri**
- Added Lable, Description, Hint text and Font Awesome Chevron to the Select component
- Added aria-label for the Select component
- Updated styles for different categories and states of the Select component
- Added the Select component to the home-page for QA

### FYI: 
For testing different **categories**: Add category="primary or plain" to lib-dropdown-input element
And for **disabled** => Add disabled to the select element

### CODE REVIEW

File: component-library/component-lib/src/lib/dropdown-input/dropdown-input.component.ts [hunk 1]
```diff
@@ -29,11 +34,12 @@ export interface IDropdownInputOptionsConfig {
 })
 export class DropdownInputComponent implements OnInit, ControlValueAccessor {
   touched = false;
+  disabled = false;
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/dropdown-input/dropdown-input.component.ts:37]: Remove

File: component-library/component-lib/src/lib/dropdown-input/dropdown-input.component.ts [hunk 2]
```diff
@@ -8,6 +8,11 @@ export interface IDropdownInputConfig {
   label?: string;
   options?: IDropdownInputOptionsConfig[]
   category?: string;
+  required?: boolean;
+  hint?: string;
+  desc?: string;
+  error?: boolean;
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/dropdown-input/dropdown-input.component.ts:14]: Remove error and disabled - they aren't used

File: component-library/component-lib/styles.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @use '~@ircc-ca/ds-sdc-core/index' as ircc-ds;
 @include ircc-ds.theme-init-required(ircc-ds.palette-journeylab(),
     default,
-    large,
+    small,
```
Comment (kristopher-houston) [inline component-library/component-lib/styles.scss:4]: Pretty sure this should still be 'large'

Comment (mahsa-ameri) [reply component-library/component-lib/styles.scss]: Pushed all changes, thanks :)

**Review by kristopher-houston — APPROVED** [review]

**Pull request merged**